code/datums/datum.dm

```diff
@@ -1,20 +1,23 @@
 /datum
-    var/gc_destroyed //Time when this object was destroyed.
-    var/list/active_timers  //for SStimer
-    var/list/datum_components //for /datum/components
-    var/ui_screen = "home"  //for tgui
-    var/use_tag = FALSE
-    var/datum/weakref/weak_reference
+
+	var/gc_destroyed //Time when this object was destroyed.
+	var/list/active_timers  //for SStimer
+	var/list/datum_components //for /datum/components
+	var/ui_screen = "home"  //for tgui
+	var/use_tag = FALSE
+	var/datum/weakref/weak_reference
+
 
 #ifdef TESTING
-    var/running_find_references
-    var/last_find_references = 0
+	var/running_find_references
+	var/last_find_references = 0
 #endif
 
 // Default implementation of clean-up code.
 // This should be overridden to remove all references pointing to the object being destroyed.
 // Return the appropriate QDEL_HINT; in most cases this is QDEL_HINT_QUEUE.
-/datum/proc/Destroy(force=FALSE, ...)
+
+/datum/proc/Destroy(force=FALSE)

```

Comment:
Don't undo this change

Reply:
Actually you should just revert all changes to this file

Reply:
Again, ask Kevinz, why he did it that way.Now it's problem of main branch.